mpd/__init__.py

```diff
@@ -27,6 +29,24 @@
 from mpd.base import ProtocolError
 from mpd.base import VERSION
 
+# MPD Protocol errors as found in CommandError exceptions
+# https://github.com/MusicPlayerDaemon/MPD/blob/master/src/protocol/Ack.hxx
+class Errors(Enum):
```

Comment:
Maybe call this `ProtocolErrorCodes` we already call Exceptions from this library MpdError.

Reply:
Yeah, I renamed it to ProtocolError, which is a bit shorter, but should be equally distinct.

Reply:
Oh, we already have ProtocolError as an exception.

Reply:
I guess it will be `FailureResponseCode`